.github/workflows/crowdin-pretranslate.yml

```diff
@@ -0,0 +1,15 @@
+# Run pre-translate with translation memory, all files/langs, at midnight daily.
+name: Crowdin Updates
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  trigger_crowdin_tm:
+    name: Crowdin Translation Memory Trigger
+    runs-on: ubuntu-latest
+    steps:
+      - uses: starship/crowdin-pretranslate-action@main
```

Comment:
I think it would be preferred to publish the action so that we can confidently version it with semver 😄

Reply:
Sure thing! I think I can just push tags and it'll create a version (we don't have to publish on the marketplace).

I'm thinking `v0.1.0` for now--a full-featured action should allow you to ignore certain files/languages as well as trigger pre-translation via machine translation (with the arguments that entails) but I don't really want to write/maintain that if nobody is going to use it.